torch/csrc/jit/codegen/cuda/executor_utils.cpp

```diff
@@ -0,0 +1,301 @@
+#include <ATen/CUDAGeneratorImpl.h>
+#include <ATen/cuda/CUDAContext.h>
+
+#include <c10/cuda/CUDACachingAllocator.h>
+
+#include <torch/csrc/jit/resource_guard.h>
+
+#include <torch/csrc/jit/codegen/cuda/kernel_resource_strings.h>
+
+#include <torch/csrc/jit/codegen/cuda/executor_utils.h>
+
+namespace torch {
+namespace jit {
+namespace fuser {
+namespace cuda {
+namespace executor_utils {
+std::string KernelPreamble() {
```

Comment:
nit: empty line here

Reply:
surprised clang format didn't do that...